101-load-balancer-ip-configured-backend-pool/azuredeploy.json

```diff
@@ -0,0 +1,430 @@
+            }
+        },
+        "vmSize": {
+            "defaultValue": "Standard_DS1_v2",
+            "type": "String",
+            "metadata": {
+                "description": "Size of the virtual machine"
+            }
+        }
+    },
+    "variables": {
+        "lbName": "[concat(parameters('projectName'),'-lb')]",
+        "lbSkuName": "Standard",
+        "lbPublicIpAddressName": "[concat(parameters('projectName'),'-lbPublicIP')]",
+        "lbPublicIPAddressNameOutbound": "[concat(parameters('projectName'),'-lbPublicIPOutbound')]",
+        "lbFrontEndName": "LoadBalancerFrontEnd",
+        "lbFrontEndNameOutbound": "LoadBalancerFrontEndOutbound",
+        "lbBackendPoolName": "LoadBalancerBackEndPool",
+        "lbProbeName": "loadBalancerHealthProbe",
+        "nsgName": "[concat(parameters('projectName'),'-nsg')]",
+        "vNetName": "[concat(parameters('projectName'),'-vnet')]",
+        "vNetAddressPrefix": "10.0.0.0/16",
+        "vNetSubnetName": "BackendSubnet",
+        "vNetSubnetAddressPrefix": "10.0.0.0/24",
+        "bastionName": "[concat(parameters('projectName'),'-bastion')]",
+        "bastionSubnetName": "AzureBastionSubnet",
+        "vNetBastionSubnetAddressPrefix": "10.0.1.0/24",
+        "bastionPublicIPAddressName": "[concat(parameters('projectName'),'-bastionPublicIP')]",
+        "vmStorageAccountType": "Premium_LRS",
+        "nicIpAddresses": [
+            "10.0.0.4",
+            "10.0.0.5",
+            "10.0.0.6"
+        ]
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Network/networkInterfaces",
+            "apiVersion": "2020-06-01",
+            "name": "[concat(parameters('projectName'),'-vm', copyIndex(1), '-networkInterface')]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('vNetName'))]",
+                "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('vNetName'), variables('vNetSubnetName'))]",
+                "[resourceId('Microsoft.Network/networkSecurityGroups', variables('nsgName'))]"
+            ],
+            "properties": {
+                "ipConfigurations": [
+                    {
+                        "name": "ipconfig1",
+                        "properties": {
+                            "privateIPAllocationMethod": "Static",
+                            "privateIpAddress": "[variables('nicIpAddresses')[copyindex()]]",
+                            "subnet": {
+                                "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('vNetName'), variables('vNetSubnetName'))]"
+                            }
+                        }
+                    }
+                ],
+                "networkSecurityGroup": {
+                    "id": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('nsgName'))]"
+                }
+            },
+            "copy": {
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
Update sort order of all items to reflect best practices